crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -5636,3 +5646,26 @@ fn test_message_eligible_for_ff() {
         assert_eq!((cond.spends[1].flags & ELIGIBLE_FOR_FF), 0);
     }
 }
+
+#[cfg(test)]
+#[rstest]
+fn test_assert_concurrent_spend_ff(#[values(true, false)] is_dedup_id: bool) {
+    let test = format!(
```

Comment:
I think it would be nice to add a comment translating the condition codes used in the test. 51, 73, 64 and 86